opta/commands/apply.py

```diff
@@ -289,7 +289,7 @@ def _apply(
                         quiet=True,
                     )
                 except CalledProcessError as e:
-                    logger.error((e.stderr or b"").decode("utf-8"))
+                    logger.error(e.stderr or b"")
```

Comment:
```suggestion
                    logger.error(e.stderr or "")
```

- Don't want to log a `bytes`
- Do we still want to log an empty string if stderr is empty? Does any useful information get logged in that case?